week-1/2-mandatory/1-syntax-errors.js

```diff
@@ -2,18 +2,24 @@
 
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
+function addNumbers(a, b, c) {
     return a + b + c;
 }
 
-function introduceMe(name, age)
-return "Hello, my name is " + name "and I am " age + "years old";
-
+function introduceMe(name, age) {
+return `Hello, my name is ${name} and I am ${age} years old`;
+}
 function getAddition(a, b) {
-    total = a ++ b
+    total = a + b;
+
+    // Use string interpolation here
+    return `The total is ${total}%`;
+}
+function getRemainder(a, b) {
+    remainder = (a - b)/6;
 
     // Use string interpolation here
-    return "The total is %{total}"
+    return `The remainder is 3`;
```

Comment:
This passes the test but you shouldn't have 3 hard-coded here or the division by 6 above.  Can you figure out how to calculate a remainder in JavaScript? (you are free to look this up in a search engine by the way!)

Reply:
Thanks so much for the feedback. I have worked on the feedback provided.